gui/wxpython/startup/guiutils.py

```diff
@@ -319,7 +319,7 @@ def delete_mapset_interactively(guiparent, grassdb, location, mapset):
                 style=wx.OK | wx.ICON_ERROR | wx.CENTRE,
             )
         dlg.Destroy()
-        return False
+    return False
```

Comment:
And what about the `dlg.Destroy()`? That still gets executed only when `dlg.ShowModal() == wx.ID_YES` but not when *Cancel* is pressed. See the reasoning and context in the previous reviews ([July 14](https://github.com/OSGeo/grass/pull/789#discussion_r454421731) and [July 16](https://github.com/OSGeo/grass/pull/789#discussion_r456174832)).

There is still the issue with the other return statement in this function I mentioned before:

> Additionally, there is another return above in this function which returns `None`, so that's inconsistent. It needs to return `False`.

It is the if-statement `if mapset == "PERMANENT":`, line with `return` only.